lib/topological_inventory/ansible_tower/parser/service_inventory.rb

```diff
@@ -15,7 +15,7 @@ def parse_service_inventory(inventory)
               "variables"                       => inventory.variables,
               "total_inventory_sources"         => inventory.total_inventory_sources,
               "inventory_sources_with_failures" => inventory.inventory_sources_with_failures,
-              "insights_credential"             => inventory.insights_credential,
+              "insights_credential"             => inventory&.insights_credential,
```

Comment:
it won't work, because the inventory exists, but method doesn't.
Try `insights_credential_id` if probably exists.

Or I'm for deleting the whole line, I doubt we are using relation inside jsonb column somewhere.

Reply:
good point, I'm pretty sure we aren't using it anywhere either (and why would we? thats what sources is for right).

Pushed up a change removing the line.